### PR TITLE
New version of W3C VC identity specification

### PR DESCRIPTION
**Pull request by scouten-adobe**
Intended to replace #60.

Hopefully clearer language for VC-based identity assertions.

### CODE REVIEW

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -428,7 +448,109 @@ The signature type is represented by the `sig_type` field and must be one of the
...
+    "https://creator-assertions.github.io/tbd/tbd"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "CreatorIdentityAssertion"
+  ],
+  "issuer": "did:example:2g55q912ec3476eba2l9812ecbfe",
+  "credentialSubject": {
+    "tbs": {
+      "referenced_assertions": [
```
Comment (Francisc) [inline docs/modules/ROOT/pages/index.adoc:530]: This should be camelCased to match the rest of the document.

File: docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -428,7 +448,109 @@ The signature type is represented by the `sig_type` field and must be one of the
...
+* The `@context` field MUST contain at least the following entries:
+** `"https://www.w3.org/ns/credentials/v2"`
+** `"https://creator-assertions.github.io/tbd/tbd"`
+* The `type` field MUST contain at least the following entries:
+** `"VerifiableCredential"`
+** `"CreatorIdentityAssertion"`
+* The `credentialSubject` field MUST contain a `tbs` entry, which MUST be the JSON serialization of the `tbs` data structure presented for signature. All bytestring entries in the `tbs` data structure (i.e. `hash` entries in hashlink URL data structures) MUST be converted to the corresponding Base-64 plain-text strings. (TO DO: Specify Base-64 encoding options.)
+* The `credentialSubject` field MAY contain other entries. The meaning of any such entries is not defined in this specification.
+* The `issuer` field MUST exist.
+* A suitable `proof` field MUST exist and prove issuance of this credential by `issuer`. (TO DO: Review https://www.w3.org/TR/vc-data-integrity/.)
```
Comment (rghiorghisor) [inline docs/modules/ROOT/pages/index.adoc:507]: In issue #73 there is an ongoing discussion what security recommendations could be. As per [4.9 Securing Mechanisms](https://www.w3.org/TR/vc-data-model-2.0/#securing-mechanisms) section, there are to options:
- enveloping proof ([DATA INTEGRITY](https://www.w3.org/TR/vc-data-integrity/))
- enveloping proof ([VC-JOSE-COSE](https://www.w3.org/TR/vc-data-model-2.0/#bib-vc-jose-cose))

File: docs/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -444,7 +464,100 @@ The signature type is represented by the `sig_type` field and must be one of the
...
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://creator-assertions.github.io/tbd/tbd"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "CreatorIdentityAssertion"
+  ],
+  "issuer": "did:example:2g55q912ec3476eba2l9812ecbfe",
+  "credentialSubject": {
```
Comment (rghiorghisor) [inline docs/modules/ROOT/pages/index.adoc:535]: I think that credentialSubject should also have a type field. This type should be defined in the @context and be used for validation purposes.

**Comment by scouten-adobe** [comment]
Closed in favor of #90.

**Pull request closed without merging**